.github/workflows/build.yml

```diff
@@ -14,7 +14,7 @@ env:
 jobs:
   build:
     name: Build
-    runs-on: ubuntu-latest
+    runs-on: macOS-latest
```

Comment:
There is a limit on mac-os agents per repo, makes sense to check of we can merge with instrumentation tests (both variants). Definitely makes sense to merge with `Publication verification`, both builds all libs, the second one just does simple job on packing and posting to local folder. It will reduce amount of unique caches. But fine to do as a separate PR.

Reply:
I would suggest instead separately into even more granular jobs instead.
i.e. a job that builds:
- Android (ubuntu)
- Web (ubuntu)
- Desktop (ubuntu)
- iOS (mac)

or at the very least split iOS build into its own job instead. This will reduce the amount of agents used overall by parallel PRs as the iOS build job would finish a lot faster.

Reply:
> I would suggest instead separately into even more granular jobs instead.

It is such a pain to set up and support it 😭 either specifying manually all platform related tasks, or relying on filtering plugins ([trust me](https://medium.com/bumble-tech/continuous-delivery-for-your-kotlin-multiplatform-library-3ab5ad5cba59), but this particular approach does not work anymore because of Gradle updates).

If you are ready to do it will support you!